mods/ts/rules/defaults.yaml

```diff
@@ -609,6 +609,17 @@
 	Tooltip:
 		Name: Palette
 
+^Tunnel:
```

Comment:
Waterfalls could also use this (booth are pretty much the same in WW logics), so `^Tunnel` might not be the best option.